app/views/admin_stats/index.html.erb

```diff
@@ -3,7 +3,7 @@
     <td class="page_header" colspan="2"><%= t("admin_stats.admin_stats") %></td>
   </tr>
   <tr>
-    <td colspan="2"><%= link_to t("admin_stats.admin"), :action => "index" %> | <%= link_to t("admin_stats.news"), :action => "news" %></td>
+    <td colspan="2"><%= link_to t("admin_stats.admin"), :action => "index" %></td>
```

Comment:
I was getting RoutingError. Couldn't find the action named news anywhere let alone AdminStatsController.